File: futsal/view/futsal.py

from django.shortcuts import render
from django.http import JsonResponse
from rest_framework.decorators import api_view
from rest_framework.response import Response

from futsal.models import Admin, Futsal, Notice


@api_view(['GET'])
def getFutsal(request):
    futsals = Futsal.objects.all()
    returnable = []
    for i in futsals:
        aDict = {"id":i.id,"name":   i.name, "latitude": i.latitude,
                 "longitude": i.longitude}
        returnable.append(aDict)
    return Response(returnable)

File: letsfutsal/lib/Entities/Futsal.dart
class Futsal{
int id;
  String name;
  double lat;
  double lng;
  Futsal(this.id, this.name, this.lat, this.lng);

  factory Futsal.fromJson(Map<String, dynamic> json){
  return Futsal(json["id"], json["name"], json["latitude"], json["longitude"]);
  }

}